static/js/tabs.js

```diff
@@ -0,0 +1,62 @@
+function setupTabs() {
+  var tabs = document.querySelectorAll(".js-tab");
+
+  tabs.forEach((tab) => {
+    const tabSelector = tab.getAttribute("href");
+    const tabContent = document.querySelector(tabSelector);
+
+    if (tabContent) {
+      tab.setAttribute("aria-selected", "false");
+      if (tabContent.id !== "tab-one__content") {
+        tabContent.classList.add("u-hide");
+      }
+
+      tab.addEventListener("click", function (e) {
+        e.preventDefault();
+        e.stopPropagation();
+        toggleTab(e.currentTarget);
+      });
+    }
```

Comment:
Can you add an else here to `console.warn` that the tab content does not exist. To help to debug in the future.